src/govuk/components/footer/footer.yaml

```diff
@@ -67,6 +67,22 @@ params:
       type: object
       required: false
       description: HTML attributes (for example data attributes) to add to the anchor in the footer navigation section.
+- name: contentLicence
+  type: string
+  required: false
+  description: HTML or text to replace the default Open Government Licence notice with.
```

Comment:
Is it potentially a slight issue that we split HTML and text into their own properties on other fields, but not here?

Reply:
I figured splitting it up might not be necessary as it's fairly likely to be HTML. There are other parameters that aren't split (like `visuallyHiddenText`), though that does include 'text' in the name. 

Would it be better named as `contentLicenceHtml` or do you think it's better to split it up?

Reply:
`copyright` is a more awkward example of this, I think.

The coat of arms is set as the background image of the crown copyright link. As such, a user cannot use HTML to set a custom link unless `hideCrownCopyrightArms: true` (which removes the link and thus the arms), in which case they can. The typical input then switches between being HTML or plain text depending on that context.